modules/nw-ingress-controller-configuration-parameters.adoc

```diff
@@ -116,6 +116,15 @@ Ciphers and the minimum TLS version of the configured security profile are refle
 
 * `WildcardsDisallowed`: Indicates only routes with a wildcard policy of `None` are admitted by the Ingress Controller. Updating `wildcardPolicy` from `WildcardsAllowed` to `WildcardsDisallowed` causes admitted routes with a wildcard policy of `Subdomain` to stop working. These routes must be recreated to a wildcard policy of `None` to be readmitted by the Ingress Controller. `WildcardsDisallowed` is the default setting.
 
+|`IngressControllerLogging`
+|* `logging` defines parameters for what should be logged where. If this field is empty, operational logs are enabled but access logs are disabled.
+* `access` describes how the client requests should be logged. If this field is empty, access logging is disabled.
+* `ContainerLoggingDestinationParameters` describes parameters for the Container logging destination type.
```

Comment:
``` `ContainerLoggingDestinationParameters` ``` → ``` `container` ```.

It might be worth noting that currently, there are no parameters for container logging, so this field should be empty or omitted.

``` `container` ``` and ``` `syslog` ``` are fields underneath the ``` `destination` ``` field.  To be consistent with other rows, ``` `logging` ``` should be in a normal paragraph, and ``` `access` ``` should be in a bulleted list below ``` `logging` ```.  Then ``` `destination` ``` should be in a second-level bulleted list below ``` `access` ```, and then ``` `type` ```, ``` `container` ```, and ``` `syslog` ``` should be in a third-level bulleted list below ``` `destination` ```.